index.html

```diff
@@ -25,6 +25,18 @@
                                 </a>
                             </li>
                         </ul>
+                        <section class="work-with-us">
+                            <h2>Want to work with us?</h2>
+                            <p>E-mail your resume to <a href="mailto:tech@cfpb.gov?subject=Interested in working in tech at CFPB">tech@cfpb.gov</a> and
+                        we'll contact you about relevant opportunities throughout the year. Previous openings have been in the fields of cybersecurity, graphic design, web development, user experience, and data architecture/analysis.</p>
```

Comment:
It's [best to add semantic line breaks](https://github.com/cfpb/front-end/blob/master/markup.md#semantic-line-breaks-in-source-code) at 80 chars here and throughout -